bootstrap.sh

```diff
@@ -35,14 +48,14 @@ install_packages_apt()
 {
     # apt update and install dependencies
     sudo apt-get update
-    sudo apt-get install python3-pip expect figlet graphviz python3-tk -y
+    sudo apt-get install python3-pip python3-venv expect figlet graphviz python3-tk -y
```

Comment:
replace python3 with $python

Reply:
The apt package names shouldn't change. `python3` here isn't the Python path, but a part of the packages' name. They are installed to system path anyway (`/usr/lib/`).

Reply:
Ack